### PR TITLE
Update docs for v2021.08.02 release

### PR DESCRIPTION
**Pull request by sakiib**
Signed-off-by: sakibalamin <sakibalamin@appscode.com>

### CODE REVIEW

**Comment by tamalsaha** [comment]
- [ ] Everything needs to run in `demo` namespace including test pods, test dbs etc.

File: docs/guides/secret-engines/mongodb/overview.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  docs_{{ .version }}:
-    identifier: overview-mongodb
-    name: Overview
-    parent: mongodb-secret-engines
-    weight: 10
+docs_{{ .version }}:
+identifier: overview-mongodb
+name: Overview
+parent: mongodb-secret-engines
+weight: 10
```
Comment (tamalsaha) [inline docs/guides/secret-engines/mongodb/overview.md:8]: Indentation can't change.

Comment (sakiib) [reply docs/guides/secret-engines/mongodb/overview.md:8]: on it!

File: docs/concepts/secret-engine-crds/database-secret-engine/elasticsearch.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
+title: ElasticsearchRole | Vault Secret Engine
+menu:
+docs_{{ .version }}:
+identifier: elasticsearch-database-crds
+name: ElasticsearchRole
+parent: database-crds-concepts
+weight: 10
```
Comment (tamalsaha) [inline docs/concepts/secret-engine-crds/database-secret-engine/elasticsearch.md:8]: Fix indentation

**Pull request merged**